third_party/cares/config_linux/ares_config.h

```diff
@@ -46,38 +45,29 @@
 /* Specifies the number of arguments to getservbyport_r */
 #define GETSERVBYPORT_R_ARGS 6
 
-/* Specifies the size of the buffer to pass to getservbyport_r */
-#define GETSERVBYPORT_R_BUFSIZE 4096
-
 /* Define to 1 if you have AF_INET6. */
-#define HAVE_AF_INET6 1
+#define HAVE_AF_INET6
 
 /* Define to 1 if you have the <arpa/inet.h> header file. */
-#define HAVE_ARPA_INET_H 1
+#define HAVE_ARPA_INET_H
 
 /* Define to 1 if you have the <arpa/nameser_compat.h> header file. */
-#define HAVE_ARPA_NAMESER_COMPAT_H 1
+#define HAVE_ARPA_NAMESER_COMPAT_H
 
 /* Define to 1 if you have the <arpa/nameser.h> header file. */
-#define HAVE_ARPA_NAMESER_H 1
+#define HAVE_ARPA_NAMESER_H
 
 /* Define to 1 if you have the <assert.h> header file. */
-#define HAVE_ASSERT_H 1
+#define HAVE_ASSERT_H
 
 /* Define to 1 if you have the `bitncmp' function. */
 /* #undef HAVE_BITNCMP */
 
 /* Define to 1 if bool is an available type. */
-#define HAVE_BOOL_T 1
+#define HAVE_BOOL_T
 
-/* Define HAVE_CLOCK_GETTIME_MONOTONIC to 1 if you have the clock_gettime
- * function and monotonic timer.
- *
- * Note: setting HAVE_CLOCK_GETTIME_MONOTONIC causes use of the clock_gettime
- * function from glibc, don't set it to support glibc < 2.17 */
-#ifndef GPR_BACKWARDS_COMPATIBILITY_MODE
-  #define HAVE_CLOCK_GETTIME_MONOTONIC 1
```

Comment:
I think we need to preserve this manual edit. `HAVE_CLOCK_GETTIME_MONOTONIC` being defined forces a dependency on glibc 1.17+ - it should break a couple of ruby distrib tests.

Reply:
Done.